components/config/examples/generate_sublime.rs

```diff
@@ -26,10 +29,31 @@ fn main() {
             builder.add_from_folder(package_dir, true).unwrap();
             let ss = builder.build();
             dump_to_file(&ss, packpath_newlines).unwrap();
-
-            for s in ss.syntaxes() {
-                if !s.file_extensions.is_empty() {
-                    println!("- {} -> {:?}", s.name, s.file_extensions);
+            let mut syntaxes: HashMap<String, SyntaxReference> = HashMap::new();
+            for s in ss.syntaxes().iter() {
+                if !syntaxes.contains_key(&s.name) {
```

Comment:
HashMaps have something called the entry api: https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html

The code would look like https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=10318e535286386fc4577c213b6fe0f1 if I had to write it I think

Reply:
Interesting :-) ! I'll use that definitely